lib/dataSchemaHelpers.ts

```diff
@@ -72,6 +72,31 @@ export interface SchemaJson extends BaseEntity {
     '@graph': SchemaData[];
 }
 
+export enum SchemaDataId {
```

Comment:
it seems weird that we're dynamically generating column but then hardcoding this?

Reply:
I disagree based on the usage - there is hardcoded stuff relating to specific data ids so its good to put it in an enum

Reply:
Right, for now we only enumerate the ids that we need to use, not all of them. We need to know the actual id when we want to customize a specific column.